### PR TITLE
Completed Trigrams Assignment

### PR DESCRIPTION
**Pull request by alexanderboone96**
I learned a lot during this one. 

General question: What is your opinion on which docstring syntax is preferred? From what I've read, there are different docstring standards that adhere to PEP 257 but I'd like to use the "industry-standard" if there is one.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "editor.rulers": [
+        80
+    ],
+    "editor.wordWrap": "on"
+}
```
Comment (natasha-aleksandrova) [inline .vscode/settings.json]: please do not change anything outside of the student folder - I need you to remove this file please.

**Comment by natasha-aleksandrova** [comment]
@alexanderboone96 good question.

The standard is to use triple double quoted strings:
```
def test():
    """This is a docstring"""
```

however, there are various formats on how you define your parameters and return etc - this will depend on the standards in your company / team. For example, my team uses Pycharm which auto generates docstring format so it makes sense for us to use that format.

File: students/alexander_boone/lesson04/assignments/mailroom/mailroom.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+import os
+import sys
+
+donors = {'Arnold Schwarzenegger': [100000, 1],
+          'Lebron James': [1000000, 1],
+          'Elon Musk': [2000000, 1],
+          'Walter White': [500000, 1],
+          'Gordon Ramsay': [1280000, 1]}
```
Comment (natasha-aleksandrova) [inline students/alexander_boone/lesson04/assignments/mailroom/mailroom.py]: 👍

File: students/alexander_boone/lesson04/assignments/mailroom/mailroom.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        options = ["1. Send a Thank You to a single donor", "2. Create a"
+                   + " Report", "3. Send letters to all donors",
+                   "4. Quit"]
+        print(f"----- Main Menu -----\n{options[0]}\n{options[1]}"
+              + f"\n{options[2]}\n{options[3]}")
+
+        # ask for and run user response
+        while int(response) not in response_dict:
+            response = int(input("Enter a number: "))
+        response_dict.get(response)()
```
Comment (natasha-aleksandrova) [inline students/alexander_boone/lesson04/assignments/mailroom/mailroom.py]: everything looks great!

File: students/alexander_boone/lesson04/assignments/mailroom/mailroom.py [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+
+# main code
+if __name__ == '__main__':
+
+    response_dict = {1: thank_you, 2: create_report, 3: letters_to_all,
+                     4: quit_program}
+    response = 0
+    while response != 4:
+        response = 0
```
Comment (natasha-aleksandrova) [inline students/alexander_boone/lesson04/assignments/mailroom/mailroom.py]: you can just have this as `while True`

File: students/alexander_boone/lesson04/assignments/mailroom/mailroom.py [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+        # display main menu with options
+        options = ["1. Send a Thank You to a single donor", "2. Create a"
+                   + " Report", "3. Send letters to all donors",
+                   "4. Quit"]
+        print(f"----- Main Menu -----\n{options[0]}\n{options[1]}"
+              + f"\n{options[2]}\n{options[3]}")
+
+        # ask for and run user response
+        while int(response) not in response_dict:
+            response = int(input("Enter a number: "))
```
Comment (natasha-aleksandrova) [inline students/alexander_boone/lesson04/assignments/mailroom/mailroom.py]: Then you can have a check if `response not in response_dict` tell the user that it's a wrong a selection.

**Comment by alexanderboone96** [comment]
Thanks for the feedback! I implemented your comments and will submit another pull request shortly.

**Comment by natasha-aleksandrova** [comment]
@alexanderboone96 please remove the extra file - we will not be able to merge

**Pull request merged**